apps/aragon-fundraising/app/src/screens/Reserves.js

```diff
@@ -221,123 +175,114 @@ export default () => {
     }
   }
 
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+
+  const editMonthlyAllocationButton = <Button icon={<img src={EditIcon} />} label="Edit monthly allocation" onClick={() => setOpened(true)}></Button>
```

Comment:
The linter prefers self-closing when the component doesn't have children `<Button />`

Reply:
Done in https://github.com/AragonBlack/fundraising/pull/114/commits/7fefdbb7c7f70f62b47cc6d0087481f72cd61c2d